ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude

```diff
@@ -14,8 +14,10 @@ THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 <#@ template debug="true" hostSpecific="true" visibility="internal" linePragmas="false"#>
 <#@ output extension=".cs" #>
 <#@ Assembly Name="System.Core.dll" #>
+<#@ Assembly Name="System.Runtime.dll" #>
 <#@ Assembly Name="System.Xml.dll" #>
 <#@ Assembly Name="System.Xml.Linq.dll" #>
+<#@ Assembly Name="System.Xml.ReaderWriter.dll" #>
```

Comment:
What APIs or classes were missing that require the addition of `System.Runtime.dll` and `System.Xml.ReaderWriter.dll`?

Reply:
Alan, please see the comment above.

Reply:
Just saw it as we commented around same time. :)